Riot/Views/RoomList/RoomCollectionViewCell.xib

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please revert your changes in this .xib (like you did in RoomCollectionViewCell.m file).
We will work on this collection view cell later